<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI2048</class>
 <widget class="QWidget" name="UI2048">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="highscoreLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Highscore:</string>
   </property>
  </widget>
  <widget class="QLabel" name="scoreLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>170</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2048</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="backToMenuButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>531</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QFrame" name="boardFrame">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>354</width>
     <height>354</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGraphicsView" name="boardView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>354</width>
      <height>354</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
